models.go

```diff
@@ -94,6 +94,13 @@ func (p OnlineQueryParams) WithStaleness(feature any, duration time.Duration) On
 	return p.withStaleness(feature, duration)
 }
 
+// WithTags returns a copy of Online Query parameters with the specified tags added.
+// For use via method chaining. See OnlineQueryParamsComplete for usage examples.
+// See https://docs.chalk.ai/docs/resolver-tags for more information on tags.
+func (p OnlineQueryParams) WithTags(tags []string) OnlineQueryParams {
+	return p.withTags(tags)
+}
+
```

Comment:
I think we've intentionally have most params passed directly when instantiating the struct, except the params that require `Feature` structs passed in. I think it could be fine passing in tags - but that would mean for consistency we have to:

1. add `withX` methods for other optional params 
2. add `withX` methods to *all* of `OfflineQueryParams, OfflineQueryParamsWithInputs, OfflineQueryParamsWithOutputs, OfflineQueryParamsComplete` which can get complex

So I think for those reasons we might not want to add withX methods for tags

Reply:
ahhh